### PR TITLE
AP-2955 Remove enable evidence upload flag

### PR DESCRIPTION
**Pull request by stephenrichards**
## Remove enable evidence upload flag

[Link to story](https://dsdmoj.atlassian.net/browse/AP-2955)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/providers/has_evidence_of_benefits/show.html.erb [hunk 1]
```diff
@@ -8,14 +8,13 @@
       ) do |form| %>
 
     <%= form.govuk_collection_radio_buttons :has_evidence_of_benefit, # attribute name
-                                            yes_no_options(yes: Setting.enable_evidence_upload? ? t('.radio_hint_yes') : nil), # collection
+                                            yes_no_options(yes: t('.radio_hint_yes')), # collection
```
Comment (chris-groves) [inline app/views/providers/has_evidence_of_benefits/show.html.erb:11]: do we need the collection comment?

File: features/support/steps_helper.rb [hunk 2]
```diff
@@ -61,9 +61,10 @@
   fill_in(name, with: value)
 end
 
-Then("I upload a pdf file") do
-  attach_file("Attach a file", Rails.root.join("spec/fixtures/files/documents/hello_world.pdf"))
-end
+# Then("I upload a pdf file") do
+#   # attach_file("Attach a file", Rails.root.join("spec/fixtures/files/documents/hello_world.pdf"))
+#   attach_file("dz-upload-button", Rails.root.join("spec/fixtures/files/documents/hello_world.pdf"))
+# end
```
Comment (chris-groves) [inline features/support/steps_helper.rb:68]: can this commented out code be deleted?

File: app/views/providers/has_evidence_of_benefits/show.html.erb (hunk 1 above)
Comment (stephenrichards) [reply app/views/providers/has_evidence_of_benefits/show.html.erb:11]: The comments at the end of each line here are the parameters as defined in the documentation of `govuk_collection_radio_buttons` - looking at the documentation [here](https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder/Builder#govuk_collection_radio_buttons-instance_method) it's quite a complicated set of parameters to pass into this method, so I think it's useful to have the parameter names as a comment

File: features/support/steps_helper.rb (hunk 2 above)
Comment (stephenrichards) [reply features/support/steps_helper.rb:68]: yep - done

**Review by chris-groves — APPROVED** [review]

**Pull request merged**